src/helpers/Environment.ts

```diff
@@ -179,20 +179,11 @@ export class EnvHelper {
   }
 
   static getZapperAPIKey() {
```

Comment:
Looks like this method is redundant since it is only returning same static string every time, @appleseed-iii

Reply:
ok moved it into zaphelper